attestation/product/product.go

```diff
@@ -45,12 +45,45 @@ var (
 func init() {
 	attestation.RegisterAttestation(Name, Type, RunType, func() attestation.Attestor {
 		return New()
-	})
+	}, attestation.StringConfigOption("includeglob", "Glob that will be used to match files to record as products", func(a attestation.Attestor, includeGlob string) (attestation.Attestor, error) {
+		attestor, ok := a.(*Attestor)
+		if !ok {
+			return nil, fmt.Errorf("unexpected attestation type %T", a)
+		}
+
+		attestor.includeGlob = includeGlob
+		return attestor, nil
+	}), attestation.StringConfigOption("excludeglob", "Glob that will be used to match files to exclude as products", func(a attestation.Attestor, excludeGlob string) (attestation.Attestor, error) {
```

Comment:
Im thinking should take priority.  Also, this will only be for subject creation when implemented, right?

Reply:
To be honest I haven't put any thought into the actual implementation of these actual settings.  It depends on the user case that's driving these options but my thought is yeah, exclude would take precedence.

Currently these two options are not implemented and that will likely belong in it's own issue, this PR is mostly focused around generally supporting configuring attestors through the CLI.